tests/SprykerSdkTest/Zed/AppSdk/Communication/Console/CreateManifestConsoleTest.php

```diff
@@ -37,6 +37,20 @@ public function testCreateManifestConsole(): void
         // Act
         $commandTester->execute([CreateManifestConsole::MANIFEST_NAME => 'Manifest', '--' . CreateManifestConsole::OPTION_MANIFEST_PATH => 'config/example/en_US.json'], ['verbosity' => OutputInterface::VERBOSITY_VERBOSE]);
 
+        // Assert
+        $this->assertSame(AbstractConsole::CODE_SUCCESS, $commandTester->getStatusCode());
+    }
+
+        /**
+         * @return void
+         */
+    public function testCreateManifestConsoleWithFileExists(): void
+    {
+        $commandTester = $this->tester->getConsoleTester(new CreateManifestConsole());
+
+        // Act
+        $commandTester->execute([CreateManifestConsole::MANIFEST_NAME => 'Manifest', '--' . CreateManifestConsole::OPTION_MANIFEST_PATH => codecept_data_dir('invalid/manifest/en_IN.json')], ['verbosity' => OutputInterface::VERBOSITY_VERBOSE]);
+
         // Assert
         $this->assertSame(AbstractConsole::CODE_SUCCESS, $commandTester->getStatusCode());
```

Comment:
You should have `CODE_ERROR` and you should also check for the returned message.

Reply:
Added a check for returned message